### PR TITLE
New topic for dashboard overview

### PR DESCRIPTION
**Pull request by rashminandasahoo**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/18_Dashboards/dashboards-overview.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- Deployments
+- Feature Flags
+- Security Tests
+
+## Important
+
+- You can create custom dashboards at the [account scope](../4_Role-Based-Access-Control/1-rbac-in-harness.md#rbac-scope) only.
+- Make sure you have the `Manage` permissions for **Dashboards** to create, edit, and delete custom dashboards.
+- Make sure you have the `View` permissions for **Dashboards** to view the custom dashboards.
+- By default, custom dashboards have data for the past 30 days.
```
Comment (Rohit-Kaliki) [inline docs/platform/18_Dashboards/dashboards-overview.md:22]: This is specific to each dashboard and metric, probably can remove this line

**Review by Rohit-Kaliki — APPROVED** [review]

File: docs/platform/18_Dashboards/dashboards-overview.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+title: Dashboards Overview
+description: An overview of dashboards in Harness.
+sidebar_position: 2
+---
+
+Harness Dashboards display key metrics and data related to your builds, deployments, security, and cloud costs. Dashboards let you track the health and performance of your deployments, monitor key metrics and KPIs, and quickly identify potential issues. You can customize the dashboards to show specific metrics and data relevant to your needs, and you will be able to access them from a central location. The Harness Dashboards provide actionable insights into continuous delivery processes, so you can achieve faster and more efficient deployments.
```
Comment (charanya-j) [inline docs/platform/18_Dashboards/dashboards-overview.md:7]: can we replace 'and you will be able to' to 'you can'

File: docs/platform/18_Dashboards/dashboards-overview.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- Make sure you have the `View` permissions for **Dashboards** to view the custom dashboards.
+
+## Navigate to Dashboards
+
+To view and access dashboards, select **DASHBOARDS** from the left navigation in your Harness account.
+You can filter the data on the dashboards by selecting specific module(s).
+
+![](./static/filter-dashboard-data.png)
+
+Once you have one or more custom dashboards available to you, the **Dashboards** page lists them.
```
Comment (charanya-j) [inline docs/platform/18_Dashboards/dashboards-overview.md:30]: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/once.
Microsoft style guide recommends to use After

File: docs/platform/18_Dashboards/dashboards-overview.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+- Go to the dashboard you want to clone.
+- Click on the three dots (...) in the top-right corner of the dashboard.
+- Select "Clone Dashboard".
+- Give the cloned dashboard a new name.
+  Optionally, choose a different folder for the cloned dashboard.
+- Click on "Clone" to create the cloned dashboard.
+ 
+:::note
+The cloned dashboard will contain all the same widgets and configurations as the original dashboard.
```
Comment (charanya-j) [inline docs/platform/18_Dashboards/dashboards-overview.md:56]: Can we say "contains" instead of "will contain"?

**Review by charanya-j — DISMISSED** [review]

File: docs/platform/18_Dashboards/dashboards-overview.md (hunk 2 above)
Comment (rashminandasahoo) [reply docs/platform/18_Dashboards/dashboards-overview.md:7]: The sentence starts with "you can" so adding it here again won't look right. However, I have modified it to "and access".

File: docs/platform/18_Dashboards/dashboards-overview.md (hunk 4 above)
Comment (rashminandasahoo) [reply docs/platform/18_Dashboards/dashboards-overview.md:56]: Done

File: docs/platform/18_Dashboards/dashboards-overview.md (hunk 1 above)
Comment (rashminandasahoo) [reply docs/platform/18_Dashboards/dashboards-overview.md]: Done

File: docs/platform/18_Dashboards/dashboards-overview.md (hunk 2 above)
Comment (charanya-j) [reply docs/platform/18_Dashboards/dashboards-overview.md]: Yeah, I missed the You can at the beginning.

**Review by charanya-j — APPROVED** [review]

**Pull request merged**